docs/circuit_cutting/tutorials/04_automatic_cut_finding.ipynb

```diff
@@ -275,7 +275,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.16"
+   "version": "3.9.6"
```

Comment:
Can probably restore this from main and remove it from the diff here

Reply:
This is not actually resolved, but I won't hold up this PR for this.